### DISCUSSION TITLE
Is there a way to check for missing environment variables?

### DISCUSSION THREAD

**Discussion by dejagersh** [symfony/symfony#49873] [Q&A]
I have a bunch of services/controllers that require environment variables that are injected as such:

`#[Autowire('%env(MY_REQUIRED_VARIABLE)%')]`

Is there a way to check if these all resolve to valid values? Right now this only errors when the service/controller is actually used. I would like our app to fail as as fast as possible.

**Comment by ker0x** [comment]
You can use `php bin/console debug:container --env-vars` to display all symfony container environment variables!
